File: web-client/src/app/domain/word-bundle.ts
export class WordBundle {
  id?: number;
  name: string;
  importance: number;
  wordIds: number[];

  static wordBundleAscNameComparator(wordBundle1: WordBundle, wordBundle2: WordBundle): number {
    if (wordBundle1.name > wordBundle2.name) return 1;
    if (wordBundle1.name < wordBundle2.name) return -1;
    if (wordBundle1.name == wordBundle2.name) return wordBundle1.id - wordBundle2.id;
  }
}

File: server/app/domain/WordBundle.scala
package domain

import play.api.libs.json.Json

case class WordBundle(
                       id: Option[Long],
                       name: String,
                       importance: Byte,
                       wordIds: Seq[Long]
                     )

object WordBundle {
  implicit val format = Json.format[WordBundle]
}
